packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -181,6 +181,27 @@ const EDiscovery = SparkPlugin.extend({
     });
   },
 
+  @waitForValue('@')
+  /**
+   * Retrieves content associated with a report
+   * @param {UUID} reportId - Id of the report which contains the content
+   * @param {UUID} offset - Start position from which to retrieve content
```

Comment:
Because this is part of an options-like object, I would document it like

```
@param {Object} options
@param {UUID} options.offset
```
and so on.